scripts/framework-applications/suspend-suppliers-without-agreements.py

```diff
@@ -88,25 +103,30 @@
     supplier_ids = get_supplier_ids_from_args(args)
     suppliers = find_suppliers_without_agreements(client, framework_slug, supplier_ids)
 
-    with open(output_dir / FILENAME, 'w') as csvfile:
-        csv_headers = ['Supplier email', 'Supplier ID', "No. of services suspended"]
-
-        writer = csv.writer(csvfile, quoting=csv.QUOTE_MINIMAL)
-        writer.writerow(csv_headers)
+    framework_name = framework["name"]
+    contract_title = get_framework_contract_title(FRAMEWORKS_PATH, framework_slug)
 
-        for supplier in suppliers:
-            supplier_id = supplier["supplierId"]
-            framework_info = client.get_supplier_framework_info(supplier_id, framework_slug)
+    for supplier in suppliers:
+        supplier_id = supplier["supplierId"]
+        framework_info = client.get_supplier_framework_info(supplier_id, framework_slug)
 
+        logger.info(f"{prefix}Suspending services for supplier {supplier_id}")
+        if not dry_run:
             # Do the suspending
             suspended_service_count = suspend_supplier_services(
                 client, logger, framework_slug, supplier_id, framework_info, dry_run
             )
 
-            if suspended_service_count == 0:
```

Comment:
I got rid of this block because I can't see how it does anything. If I'm wrong let me know and I'll put it back